File: HackerRank/C++/Easy/E0022.cpp
/*
Problem Statement: https://www.hackerrank.com/challenges/rectangle-area/problem
*/

class Rectangle {
protected:
    int width, height;

public:
    void display() {
        cout << width << " " << height << endl;
    }
};

class RectangleArea : public Rectangle {
public:
    void read_input() {
        cin >> width >> height;
    }

    void display() {
        cout << width * height << endl;
    }
};

File: HackerRank/Python/Easy/E0022.py
# Problem Statement: https://www.hackerrank.com/challenges/alphabet-rangoli/problem

import string

def print_rangoli(N):
	lines = []
	for i in range(N):
		line = string.ascii_lowercase[i:N]
		line = line[-1:0:-1] + line
		line = '-'.join(line).center(4*N - 3, '-')
		lines.append(line)
	lines = lines[-1:0:-1] + lines
	print('\n'.join(lines))